### PR TITLE
Added Missing Alt Value for Footer Archive Logo

### LINKED ISSUE #4584: Missing Alt Value for Footer Archive Logo Img
### Evidence / Screenshot (if possible)
image

### Steps to Reproduce

1. Go to Footer of openlibrary.org
2. Check the alt value of footer using Inspect Element Feature

* Actual: alt = ""
* Expected: Some Alt Value

### Details

- **Logged in (Y/N)?**  N
- **Browser type/version?** Chrome
- **Operating system?** Windows
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

Add an Alt Value to the Img

### Related files

[nav_foot.html](https://github.com/internetarchive/openlibrary/blob/a61250b4ade345430e406d99eb04ae102713899a/openlibrary/templates/lib/nav_foot.html)

### PR DESCRIPTION
**Pull request by dwaipayan05**
Closes #4584 

Added a missing Alt Value for the Footer Archive Logo

### CODE REVIEW

**Comment by Yashs911** [comment]
Not an issue according to @bpmcneilly (https://github.com/internetarchive/openlibrary/issues/4521#issuecomment-772730099)

**Comment by jdlrobson** [comment]
> Not an issue according to @bpmcneilly ([#4521 (comment)](https://github.com/internetarchive/openlibrary/issues/4521#issuecomment-772730099))

what about people using screen readers?

File: openlibrary/templates/lib/nav_foot.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h2>$:_('Help')</h2>
     </div>
     <hr>
     <div id="footer-details">
-      <img id="archive-logo" src="/static/images/pantheon.png" alt="" aria-hidden="true">
+      <img id="archive-logo" src="/static/images/pantheon.png" alt="Archive Logo" aria-hidden="true">
```
Comment (jdlrobson) [inline openlibrary/templates/lib/nav_foot.html:53]: Internet Archive Logo or archive.org logo might be more appropriate here. Will someone using a screenreader know what "Archive" means?

Comment (dwaipayan05) [reply openlibrary/templates/lib/nav_foot.html:53]: Would "Internet Archive Logo" work ?

**Comment by jdlrobson** [comment]
> > Not an issue according to @bpmcneilly ([#4521 (comment)](https://github.com/internetarchive/openlibrary/issues/4521#issuecomment-772730099))
> 
> what about people using screen readers?

Sorry sent ealry before I could finish by longer reply.

Expanding on this I'm super curious @bpmcneilly  on your answer there. I can understand it's not necessary given the text to the right, but it's focusable so if I was using a screen reader and focused on the image, I would be told no alternate text have no idea what I'm looking at.  How do I know that it's not necessary? Wouldn't an explict empty alt tag be better there, so I know it's intentionally been removed? Have recommendations changed since https://www.w3.org/WAI/tutorials/images/decorative/ ?

File: openlibrary/templates/lib/nav_foot.html (hunk 1 above)
Comment (jdlrobson) [reply openlibrary/templates/lib/nav_foot.html:53]: Note, best to wait for @bpmcneilly though as this answer may change :)

Comment (dwaipayan05) [reply openlibrary/templates/lib/nav_foot.html]: Yess!

**Review by jdlrobson — APPROVED** [review]
Merge pending whether @bpmcneilly says an empty alt is preferred here.

**Comment by jdlrobson** [comment]
Now I feel a bit stupid.. looks like we already had the explicit empty alt value (I'm not sure where I got the idea we had no alt attribute at all, which makes sense given https://github.com/internetarchive/openlibrary/issues/4521#issuecomment-
772730099 and https://www.w3.org/WAI/tutorials/images/decorative/ )

As a result there is no reason to change the status quo.

Hope that makes sense @dwaipayan05 - let me know if you need any explanation of the above link!

**Comment by bpmcneilly** [comment]
Sorry for the delay in responding to this, @jdlrobson! 

For posterity, I'll also add an additional note on images - decorative images (discussed in the W3C link above) can just have null `alt` attributes (`alt=""`), which will cause them to not be announced by screen readers. However, in addition to that, even if we did update the alt value, the `aria-hidden` attribute on the image would keep it hidden from assistive technologies anyway! Right now it's technically doubly hidden 😀

**Comment by jdlrobson** [comment]
> Sorry for the delay in responding to this, @jdlrobson!
> 
> For posterity, I'll also add an additional note on images - decorative images (discussed in the W3C link above) can just have null `alt` attributes (`alt=""`), which will cause them to not be announced by screen readers. However, in addition to that, even if we did update the alt value, the `aria-hidden` attribute on the image would keep it hidden from assistive technologies anyway! Right now it's technically doubly hidden 😀

No worries - I managed to confuse myself, and originally thought we were omiting the alt tag entirely, but what you write here is consistent with my understanding of how this should all work, so thanks for the reassurance/knowledge refresher. :)  Glad to have you looking at this codebase!

**Pull request closed without merging**